### DISCUSSION TITLE
Pybricks User Manual

### DISCUSSION THREAD

**Discussion by kubotafamily** [pybricks/support#1765] [Q&A] [answered]
I'm planning to participate in WRO using Pybricks with SPIKE. I need to submit an user manual. If you have one, could you please send it to me?

**Comment by dlech** [comment, marked as answer]
Do you need the manual for SPIKE Prime or for Pybricks?

You can download a PDF version of the Pybricks documentation from https://docs.pybricks.com/_/downloads/en/stable/pdf/.

**Comment by kubotafamily** [comment]
Thank you!
I need the Pybricks documentation.
But this URL gives me a 404 Not Found error...
Can you give me the correct URL?

**Reply by dlech** [reply]
Apparently the trailing `/` is important. I fixed the link.

**Reply by kubotafamily** [reply]
Thank you so much!
I'll do my best to aim for the world championships with my SPIKE and your Pybricks!
